docs/Concepts/Pruning.md

```diff
@@ -1,25 +1,61 @@
 ---
-description: Pruning
+description: Full and archive node types
 ---
 
-# Pruning
+# Full and archive node types
+
+Besu supports two node types, commonly referred to as _full nodes_ and _archive nodes_.
+
+Full nodes have just the current state of the blockchain so cannot serve the network with all data
+requests (for example, the balance of an account as at an old block). Full nodes can guarantee the
+latest state for the blockchain (and some older states, but not all). You can check current
+balances, sign and send transactions, and look at current Dapp data.
+
+Archive nodes have all of this and they also store the intermediary state of every account and
+contract for every block since the genesis block. With an archive node you can do everything you
+do with a full node, as well as access historical state data.
+
+Archive nodes require significantly more disk space (approximately 3TB) than full nodes
+(approximately 750GB).
+
```

Comment:
Maybe one thing we should add is that it is possible to re-generate all the data for an archive node using a full node (it's long, inconvenient, etc., but the data is not *lost*).

Reply:
What is the process for this? If we say you can do it, we should document how.

Reply:
It's not trivial, not something that most people will want to do (easier to just sync an archive node), but it just speaks to the security properties of a full node. Thinking about it again, I'm not sure how relevant it is here 😅

Reply:
OK. We'll leave it as is. Thanks!